src/Dumbo.php

```diff
@@ -29,6 +29,20 @@ class Dumbo
     /** @var array<callable> */
     private $middleware = [];
 
+    /** @var array<string, mixed> Variables stored in the context */
+    private $configs = [];
```

Comment:
Can we call this `variabls` instead`?

```suggestion
    private $variables = [];
```